pkg/crc/machine/start.go

```diff
@@ -482,6 +486,11 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 	}, nil
 }
 
+func deleteMCOLeaderLease(ocConfig oc.Config) error {
+	_, _, err := ocConfig.RunOcCommand("delete", "-n", "openshift-machine-config-operator", "cm", "machine-config-controller")
+	return err
+}
+
```

Comment:
@guillaumerose does it make sense to have it part of snc repo?

Reply:
yes but I think we would need to stop machine-config-controller container before doing this. 
And we still need to have this somewhere for the generated bundle. 

As you prefer.

Reply:
@guillaumerose yes it will need for generated bundle also.

Reply:
@guillaumerose can we make it part of cluster package so we can also add the `waitForResource` functionality because most of time in the CI we observed even after apiserver responding correctly, it again go back to reconcile mode and these kind of command does fails.

Reply:
yes :+1: